### PR TITLE
app-content-pages: Add stories and RTL unit tests for AboutHeader

### PR DESCRIPTION
**Pull request by goplayoutside3**
Continuation of unit tests refactor

## Package
app-content-pages

## Describe your changes

- Added a story for AboutHeader
- Converted AboutHeader's unit tests to RTL
- Added translation keys for AboutHeader link labels
- Deleted unneeded enzyme tests for the shared layout component (that component is tested in TeamsContainer and PublicationsContainer)

## How to Review

Run `yarn bootstrap` on this branch. Make sure all tests are passing on Github and locally.

Run `yarn storybook` for app-content-pages to see the new stories.

Run `yarn dev` on this branch, and visit https://local.zooniverse.org:3000/about/team (or about/publications). Try out the teams filter in the side bar to check that the feature works as expected. Can compare to production: https://www.zooniverse.org/about/team.

The only components left with enzyme tests in app-content-pages at this point are Head and AuthModal. Because those are mostly duplicated across app-project, future PRs to refactor their tests will include both FEM packages.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can sign-in and sign-out

## Refactoring
- [ ] The PR creator has described the reason for refactoring
- [ ] The refactored component(s) continue to work as expected

### CODE REVIEW

File: packages/app-content-pages/src/shared/components/TwoColumnLayout/TwoColumnLayout.js [hunk 1]
```diff
@@ -3,44 +3,24 @@ import { node } from 'prop-types'
```
Comment (goplayoutside3) [inline packages/app-content-pages/src/shared/components/TwoColumnLayout/TwoColumnLayout.js:3]: Just linting in this file.

**Review by shaunanoordin — APPROVED** [review]
## PR Review

Package: `app-content-pages`

This PR updates the AboutHeader, implementing translations and updating component tests. Also, is this the first Storybook story added for app-content-pages, I think?

- Behaviour change: AboutHeader now uses translations
- **Storybook** system added
  - Story added: AboutHeader (❔ Note: this story flags an accessibility violation, "Elements must meet minimum color contrast ratio thresholds", but this is a problem for another PR)
- Tests updated: AboutHeader, NavLink
- Tests removed: TwoColumnLayout's basic "is this rendering?" test

Code changes make sense, and I don't see any issues with the updates to the tests. Storybook also looks A-OK.

LGTM 👍

**Review by shaunanoordin — APPROVED** [review]
## PR Review

Package: `app-content-pages`

This PR updates the AboutHeader, implementing translations and updating component tests. Also, is this the first Storybook story added for app-content-pages, I think?

- Behaviour change: AboutHeader now uses translations
- **Storybook** system added
  - Story added: AboutHeader (❔ Note: this story flags an accessibility violation, "Elements must meet minimum color contrast ratio thresholds", but this is a problem for another PR)
- Tests updated: AboutHeader, NavLink
- Tests removed: TwoColumnLayout's basic "is this rendering?" test

Code changes make sense, and I don't see any issues with the updates to the tests. Storybook also looks A-OK.

LGTM 👍

**Comment by shaunanoordin** [comment]
Wha-??? Sorry, no idea why my review got double-posted!

**Pull request merged**